ci/tests/vmachine/vmachine_test.py

```diff
@@ -136,10 +136,19 @@ def vms_with_fio_test():
         machine_name = "machine-{0}".format(vm_number)
         create_machine_from_existing_raw_disk(machine_name, vpool['name'], "disk-{0}".format(vm_number))
 
-    time.sleep(30)
+    timeout = 30
```

Comment:
I'd extract all these timeouts and put these on top as constants:
TIMEOUT=30
TIMER_STEP = 5
...